tests/TestUtilities.py

```diff
@@ -47,7 +47,6 @@
     'JavaScript Object Notation',
     'JavaServer Pages',
     'Jinja2',
-    'JSON',
```

Comment:
why this change removing `JSON`?  How is this relevant to your change?

And please never rebase using a merge, like you did in https://github.com/coala/coala/pull/5712/commits/8a71a767691f8c .  It is ugly.  Now this PR has three commits.

Reply:
Because I was getting an assertion error in 
CollectorsTest.py : test_get_all_languages_without_unknown 
which said that 'JSON' is present in 'set(LANGUAGE_NAMES)' but not in 'languages '.

Reply:
Also,you already have 'JavaScript Object Notation' as a __qualname__ .So I don't think there is any need of having  JSON in TestUtilities.py when you already have 'JavaScript Object Notation' sitting there.

Reply:
If you look at travis, the tests are passing without that change.... I see you have removed it now.  Thanks.